<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visualizza_tavoli</class>
 <widget class="QMainWindow" name="visualizza_tavoli">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>480</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Ristorante Emilia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(Progetto/foto.jpg);
background-position: center;
background-repeat: no-repeat;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="indietro">
    <property name="geometry">
     <rect>
      <x>267</x>
      <y>380</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        border: 2px solid #5A5A5A;
        border-radius: 10px;
        padding: 5px;
        background-color: #F0F0F0;
        color: #5A5A5A;
    }
    QPushButton:hover {
        background-color: #D3D3D3;
    }
    QPushButton:pressed {
        background-color: #A9A9A9;
    }</string>
    </property>
    <property name="text">
     <string>Indietro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>232</x>
      <y>40</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>36</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Lista tavoli</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calendario_but">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>110</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        border: 2px solid #5A5A5A;
        border-radius: 10px;
        padding: 5px;
        background-color: #F0F0F0;
        color: #5A5A5A;
    }
    QPushButton:hover {
        background-color: #D3D3D3;
    }
    QPushButton:pressed {
        background-color: #A9A9A9;
    }</string>
    </property>
    <property name="text">
     <string>Calendario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>110</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Giorno</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_giorno">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>110</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	color: rgb(0, 0, 0);
                border: 2px solid #5A5A5A;           /* Colore del bordo */
                border-radius: 10px;                  /* Arrotondamento degli angoli */
                padding: 5px;                         /* Padding interno */
                background-color: #F0F0F0;            /* Colore di sfondo */
            }</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="tavoli_list">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>494</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    background-color: #D3CFC4;
    color: black;
    border: 1px solid black;  /* Bordo del pulsante */
}
</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pranzo_check">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>160</y>
      <width>110</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	color: rgb(0, 0, 0);
                border: 2px solid #5A5A5A;
                border-radius: 10px;
                padding: 5px;
                background-color: #F0F0F0;
            }
            QCheckBox::indicator {
                width: 15px;
                height: 15px;
                border-radius: 5px;
                border: 2px solid #5A5A5A;
                background-color: #F0F0F0;
            }
            QCheckBox::indicator:checked {
                background-color: #E4B85A;
                border: 2px solid #5A5A5A;
            }</string>
    </property>
    <property name="text">
     <string>Pranzo</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cena_check">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>160</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Apple Chancery</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
	color: rgb(0, 0, 0);
                border: 2px solid #5A5A5A;
                border-radius: 10px;
                padding: 5px;
                background-color: #F0F0F0;
            }
            QCheckBox::indicator {
                width: 15px;
                height: 15px;
                border-radius: 5px;
                border: 2px solid #5A5A5A;
                background-color: #F0F0F0;
            }
            QCheckBox::indicator:checked {
                background-color: #E4B85A;
                border: 2px solid #5A5A5A;
            }</string>
    </property>
    <property name="text">
     <string>Cena</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conferma_but">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
        border: 2px solid #5A5A5A;
        border-radius: 10px;
        padding: 5px;
        background-color: #F0F0F0;
        color: #5A5A5A;
    }
    QPushButton:hover {
        background-color: #D3D3D3;
    }
    QPushButton:pressed {
        background-color: #A9A9A9;
    }</string>
    </property>
    <property name="text">
     <string>Conferma</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
